department-of-veterans-affairs/vets-website
# Fix distance display for chosen CC provider

## Description
Fixes a bug where if a user switches the sort method to "distance from current location" and chooses a provider, the distance displayed would be from residential address, and not current location.

### How to test:
1. Go through the CC flow to the CC preferences page
2. Choose Add a provider
3. Sort by current location
4. Making note of the distance, choose closest provider
5. The distance of the chosen provider in the gray box should match the distance from step 4

## Testing done
Local and unit

## Screenshots
### Distance from residential address
![image](https://user-images.githubusercontent.com/786704/106836896-d57aa900-664e-11eb-9f04-9b3a5ede4503.png)

### Distance from current location
![image](https://user-images.githubusercontent.com/786704/106836921-df9ca780-664e-11eb-80eb-6d9874dc77e3.png)


## Acceptance criteria

## Definition of done
